src/tools/state/getCheckJobStatus.ts

```diff
@@ -13,6 +13,7 @@ import { RunningJob, FinishedJob } from '../types/toolsJob';
 import { Status } from '../types/toolsStatuses';
 import { BlastResults } from '../blast/types/blastResults';
 import { JobTypes } from '../types/toolsJobTypes';
+import { IDMappingTarget } from '../id-mapping/types/idMappingServerParameters';
```

Comment:
Not needed (not sure why eslint isn't complaining though, is this file not checked?)

Reply:
Not sure why that happened either...
Removed now